.stylelintrc.yml

```diff
@@ -63,27 +63,27 @@ rules:
     selector-combinator-space-before: always
     selector-id-pattern: null
     selector-max-specificity: null
-    selector-no-attribute: null
-    selector-no-combinator: null
-    selector-no-id: null
+    #selector-no-attribute: null
+    #selector-no-combinator: null
+    #selector-no-id: null
     selector-no-qualifying-type: null
-    selector-no-type: null
-    selector-no-universal: true
+    #selector-no-type: null
+    #selector-no-universal: true
```

Comment:
Hiding issues from the logs doesn't mean they aren't there :see_no_evil:

Reply:
Hey Christian, I know commenting does not solve the issue and this pull request is not ready to be merged yet. I am looking for solutions & have found that some of the rules are not existing in our project under stylelint node module and their configurations are still in place in .stylelintrc.yml. While running the "npm run lint" travis ci was showing the error "Error: Undefined rule selector-no-attribute" for every rules which are not in our project. So meanwhile, I commented these rules to check if we are facing any other issue or not & am trying to resolve it but please suggest if you have any idea regarding the same. Below is one of the reference link which suggest to remove the unused rules from stylelint config: https://stackoverflow.com/questions/45285437/updated-stylelint-undefined-rule-selector-no-id

Reply:
Changelogs have a wealth of information, each of the rules was renamed.
See:
https://github.com/stylelint/stylelint/blob/master/CHANGELOG.md#780
and 
https://github.com/stylelint/stylelint/blob/master/CHANGELOG.md#7120

For a long term solution to handling upgrades, preset configs: recommended and/or standard, are a pretty good idea:
https://github.com/stylelint/stylelint-config-recommended
https://github.com/stylelint/stylelint-config-standard

Reply:
Thank you. I have applied the changes. Please check and suggest if anything is wrong.